students/1st/feature:1st:hayounggam/js/likeBtn.js

```diff
@@ -0,0 +1,14 @@
+const likeBtns = document.querySelectorAll(".js-like-btn");
+
+const toggleLikeBtns = (event) => {
+  const { target } = event;
+  target.classList.toggle("fas");
+}
```

Comment:
구조분해할당 굳굳! 👍 
굳이 변수를 선언하지 않고도 할 수 있을 것 같네요.
```suggestion
const toggleLikeBtns = (event) => {
  event.target.classList.toggle("fas");
}
```

Reply:
지적 감사합니다!!
구조분해할당을 안써주는게 코드가 더 간결했네요😂
코드 수정 완료했습니다!

https://github.com/wecode-bootcamp-korea/webucks-javascript/pull/9/commits/9c3390b648f4247caae605433935e1d7f081c089